Tests/TuistSupportTests/Utils/GlobTests.swift

```diff
@@ -0,0 +1,360 @@
+import Foundation
+import XCTest
+@testable import TuistSupport
+
+//
+//  Created by Eric Firestone on 3/22/16.
+//  Copyright © 2016 Square, Inc. All rights reserved.
+//  Released under the Apache v2 License.
+//
+//  Adapted from https://gist.github.com/blakemerryman/76312e1cbf8aec248167
```

Comment:
Is this still true? To me, the code below doesn't seem copy-pasted. If there are substantial changes, I'd only keep the following note:
```suggestion
//  Inspired by: https://gist.github.com/efirestone/ce01ae109e08772647eb061b3bb387c3
```

Reply:
LGTM